charter.md

```diff
@@ -63,6 +63,30 @@ are able to proceed with a meeting.
 * Quorum to meet is 1/2 members attending (eg: 6/12, 6/11, 4/7)
 * Quorum to vote in a meeting is 2/3 members attending (eg: 8/12, 8/11, 5/7)
 
+## Elections
+
+Every year, the Steering Committee holds an election for open seats with SIG 
+Contributor Experience owning the process. Our [election policy document](https://git.k8s.io/steering/elections.md) 
+covers the details for how this works.
+### Vacancies and Removal
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
+filled. 
+
+In case this fails to fill the seat, a special election for that position will 
+be held as soon as possible. [Eligible voters](https://git.k8s.io/steering/elections.md#eligibility-for-voting) from the most recent election 
+will vote in the special election (ie: eligibility will not be redetermined at 
+the time of the special election). A committee member elected in a special 
+election will serve out the remainder of the term for the person they are 
+replacing, regardless of the length of that remainder.
+#### Removal
+
+A Steering Committee member may be removed by a two-thirds vote of the other 
```

Comment:
Generally speaking things like this have some sort of defined initiation procedure, not just a resolution procedure. I would expect this to require some sort of public charge and some sort of minimum amount of time for a defense. I would expect e.g. at least two board members to agree on the charge before this resolution procedure is triggered.

Reply:
That's a very powerful ability to give to the committee, and I think unanimity (sans person in question) would be a better bar than 2/3 rounding up.

I also think giving the person in question zero warning and zero time to even prepare a statement is not fair at all.

I would much prefer a charge + time + vote than this. If it's worth removing someone, it's worth a public spectacle, you get a lot more "this is who we are" milage out of that. And IMO if we're not willing to pay the cost of public drama then we don't actually want the person to be removed that badly. Charges can be vague if necessary.

Reply:
Policies like this have to work for the good of the community both when used well and when used poorly. If we're ejecting someone for good reasons the community should appreciate the board's actions. If we're ejecting someone for bad reasons the community should be angry and vote out the board at the next election. The community needs something to evaluate in order to be a good check on the board's behavior.

Reply:
Do we have prior art from other communities, and how it has worked for them?

This does seem like an extraordinary power to give the committee. It is the power to effectively undo the results of the election for a specific member. I can see reasons to remove someone - for example, code of conduct violation or dereliction of duty (e.g., not showing up to any meetings or participating). But without any criteria or even guidance in the appropriate use of this power, we risk silencing voices simply because they disagree with or are unpopular among the other steering committee members. This will lead to group think and is not a healthy state for our leadership and governance organizations.

Another option to achieve the same goal could be to allow the steering committee to initiate a recall election. This would put the decisions in the hands of the community, rather than in the small number of people on the steering committee.

In either case (removal or initiating a recall), given the small size of our steering committee, I believe a more appropriate committee voting threshold would be unanimity, excluding of course the subject of the vote and any abstentions.

Reply:
I tend to agree with Daniel here, though I do not think we want a long, drawn out "hearing" with cross-examination etc.  Transparency is crititcal to trust.

I think "rationale for removal" + short time + "rebuttal" is fair.

If the target wants to avoid the attention and thinks they are likely to lose the vote, resignation is always an option.

Reply:
@thockin The crux say if a person thinks what they are doing is totally right and completely oblivious of their effect on the community. They will stand their ground rendering the steering ineffective. What you and others are suggesting prolongs the agony and the harm.

NOTE that this is NOT a process to get someone out of the community. Just from their leadership position in steering. Nothing else. When the steering members do not have confidence in this person repeating a behavior and further expand the harm they are committing (And refusing to resign) there needs to be a mechanism to quickly deal with it.

There is MUCH MORE stuff we have to deal with as a community. This was not needed at this point in time. Unfortunately here we are trying to draft something because we had total confidence in abilities of those who we vote in to leadership positions and we have been throughly let down.

This is NOT something that happens every release cycle or year. this is a very drastic step and is not to be done lightly.

Reply:
"very drastic and [won't] be done lightly" is in some tension with the (hopefully I misread, but) apparent desire to do this without letting the community at large know what is going on.

Reply:
> What you and others are suggesting prolongs the agony and the harm.

I don't buy this.  Such an action DESERVES consideration and "cooler heads".  It needs to be (relatively) dispassionate.  I'm not saying this is a 6 month or even 6 week process.  I'm talking about a week or two.

> we have been throughly let down.

This statement is what I want to be careful of.  It is not dispassionate.  I don't think the  _process_ should be crafted in response to an individual, urgent situation, especially while heads are hot.  And look, you KNOW I am not saying "do nothing".  I'm just arguing that the caution of the process should be proportional to the gravity of the result.

> this is a very drastic step and is not to be done lightly.

Exactly.  I'd rather err on the side of caution and transparency and "fairness".  Just try to turn the situation around and imagine yourself in the hotseat.

Reply:
@thockin @lavalamp ACK. will try to see what we could be doing in this process better. thanks for your input.

Reply:
@thockin @lavalamp feels like steering would write something up, give it a time period, allow conversations to be had in the time in between and then VOTE.

The conversations would help the steering leaders to convince them on which way they should vote. 

How does that sound? May i suggest one week time frame?

Reply:
I'll leave it to the PR author Paris to see if this is something that can be folded into this document.

Reply:
The community should be informed at the beginning of the time period, so at the very least they can email opinions to steering members. I think public comments would be even better.

I was going to say a week seems OK, but it'd really suck to have a board pull this on a member who is on vacation or something and can't respond. Personally I'd probably go longer, maybe 2 weeks.

Reply:
@lavalamp feels reasonable to me. I'd stick to private feedback mechanism, just email steering private so everyone on steering sees it including the person in question.

yes you are right on holidays/vacations ... so 2 weeks.

Reply:
I can live with that. But let me point out that public comments create [common knowledge](https://en.wikipedia.org/wiki/Common_knowledge_(logic)), and private ones don't. That is, it's important for the community to know what the community thinks, not just for the steering committee. 

I do see the potential for abuse with public comments, but also we do have moderation (which seems very effective, thank you moderators!).

Reply:
thanks @lavalamp